### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 1 - Esercizio 23

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#479] [Esercizi M-Z]
![foglio 1 es.23](https://github.com/user-attachments/assets/d06c074b-cfd7-4f8f-adc8-c8b163f0684c)

**Comment by marcoseverini** [comment]
<img width="671" alt="immagine" src="https://github.com/user-attachments/assets/ff280380-d66b-4bd4-8cc2-d4cb502484a1">
<img width="672" alt="immagine" src="https://github.com/user-attachments/assets/f4d6061f-c2ef-493f-a7e7-b21f14efdee6">
